opencsp/common/lib/csp/Tower.py

```diff
@@ -0,0 +1,141 @@
+"""
+Tower Class
+
+Copyright (c) 2021 Sandia National Laboratories.
```

Comment:
We don't need the copyright line here because we have a copyright.txt file in the root of the project.

Reply:
Saw the email thread! Removed the copyright line in all my files.